ansible/playbooks/backup_monitoring.yml

```diff
@@ -9,13 +9,16 @@
   tasks:
     - when: specification.components.monitoring.enabled | default(false)
       block:
-        - include_vars:
+        - name: Include prometheus vars
+          include_vars:
             file: roles/prometheus/vars/main.yml
             name: component_vars
-        - import_role:
+        - name: Run prometheus backup tasks
+          import_role:
             name: backup
             tasks_from: monitoring_prometheus_snapshot
-        - import_role:
+        - name: Run prometheus archive tasks
```

Comment:
Backup here won't be better? Or adding a comment why we are using archive there?

Reply:
sure, I will rename it!